lib/update/cluster/builder_test.go

```diff
@@ -391,113 +391,50 @@ func (s *PlanSuite) TestPlanWithIntermediateRuntimeUpdate(c *check.C) {
 	})
 }
 
-func (s *PlanSuite) TestUpdatesEtcdFromManifestWithoutLabels(c *check.C) {
```

Comment:
Removed the test about assumed etcd version since it is expected that we have both installed and upgrade versions available.